environment.yml

```diff
@@ -17,10 +17,17 @@ dependencies:
   - flake8-rst>=0.6.0,<=0.7.0
   - gitpython
   - hypothesis>=3.82
+  - ipywidgets
   - isort
   - moto
   - mypy
+  - nbconvert>=5.4.1
```

Comment:
I would put some comments and/or group dependencies in this file as much as possible

Reply:
I agree, didn't think about additional grouping which makes sense, but was thinking on a comment for each dependency to explain where/why is needed. Otherwise will be difficult to know when any can be removed.

May be in a separate PR, since all them need it, and it's unrelated to the changes here? I'll create an issue, we have a sprint tonight where surely someone can work on it.

Reply:
sure that's fine; I think grouping with a single comment is prob good enough

Reply:
I opened #26659